package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "three",
-  "version": "0.116.1",
+  "version": "0.117.0",
```

Comment:
Please remove all changes to `package-lock.json` from your PR.

Reply:
Ready. See ["version": "0.116.1"](https://github.com/mrdoob/three.js/pull/19495/commits/3c240424b262a8ac03403b31edf8e7be3e65c1b4)